nearcore/src/config.rs

```diff
@@ -1071,6 +1071,11 @@ pub fn init_configs(
         _ => {
             // Create new configuration, key files and genesis for one validator.
             config.network.skip_sync_wait = true;
+
+            // Make sure node tracks all shards, see
+            // https://github.com/near/nearcore/issues/7388
+            config.tracked_shards = vec![0];
```

Comment:
Huh, this is confusing. The way I'm reading this, this only sets the configuration to track shard-id=0, and the comment above looks more like a TODO, rather than a description of what this line does…?

Reply:
https://github.com/near/nearcore/blob/6f324e84a7a7162956f0b9985094ee146919f5ae/chain/epoch-manager/src/shard_tracker.rs#L25

Reply:
We should absolutely burn it with fire. Now may actually be a good time to do so with the incoming chunk only producers.